tap_mailchimp/schemas/automations.json

```diff
@@ -286,8 +286,32 @@
         "runtime": {
           "type": [
             "null",
-            "string"
-          ]
+            "object"
+          ],
+          "additionalProperties": true,
+          "properties":{
+            "hours":{
+              "type": [
+                "null",
+                "object"
+              ],
+              "additionalProperties": true,
+              "properties":{
+                "type":{
+                  "type":[
+                    "null",
+                    "string"
+                  ]
+                }
+              }
+            },
+            "days":{
```

Comment:
In the API doc the datatype is list of string. Can you please verify? Reference: https://mailchimp.com/developer/marketing/api/automation/list-automations/

Reply:
Updated the datatype to list of objects